kotest-assertions/kotest-assertions-json/src/jvmMain/kotlin/io/kotest/assertions/json/keys.kt

```diff
@@ -28,17 +32,27 @@ fun containJsonKey(path: String) = object : Matcher<String?> {
          else -> if (value.length < 50) value.trim() else value.substring(0, 50).trim() + "..."
       }
 
-      val passed = value != null && try {
-         JsonPath.read<String>(value, path)
-         true
-      } catch (t: PathNotFoundException) {
-         false
-      } catch (t: InvalidPathException) {
-         throw AssertionError("$path is not a valid JSON path")
-      }
-
-      return MatcherResult(
-         passed,
+      return value?.let {
```

Comment:
Perhaps doing an early return for null would make this code simpler?

Reply:
> Perhaps doing an early return for null would make this code simpler?

yeah. short circuited earlier. good call.